ebuild-maintenance/maintenance-tasks/text.xml

```diff
@@ -60,9 +60,8 @@ examples.
 </p>
 
 <p>
-When committing to git, all developers should use <c>repoman commit</c>
-instead of <c>git commit</c> to submit their ebuilds.  Before committing,
-please run <c>repoman full</c> to make sure you didn't forget something.
+Before committing, please run <c>repoman full</c> to make sure you didn't forget
+something.
```

Comment:
I think we should keep `repoman commit` here, when `repoman full` is mentioned in the same context.

Reply:
`repoman full` has the added value of verifying ebuilds. `repoman commit` has no advantage over `git commit`, besides teaching developers not to learn git.

Reply:
Huh, but `repoman commit` does a verification step first:
> **commit**
> Scan directory tree for QA issues; if OK, commit via VCS

Reply:
…and the original tip said to run both. In any case, this is not the place to discuss committing tools. If you prefer, I can make that into `repoman full` OR `repoman commit`.

Reply:
`repoman commit` will add the tag line to the commit message, and I believe this is what we generally recommend?

So, I'd just leave the first sentence in place, and replace the second sentence by a short explanation that `repoman full` can be used to get a longer listing.